### PR TITLE
Fix left sliders

### PR DESCRIPTION
**Pull request by ThislsAUsername**
This was really bugging me lol

### CODE REVIEW

File: src/UI/Art/Animation/BaseUnitActionAnimation.java [hunk 1]
```diff
@@ -4,9 +4,7 @@
 import java.util.ArrayList;
 
 import Engine.XYCoord;
-import UI.Art.SpriteArtist.SpriteLibrary;
-import UI.Art.SpriteArtist.SpriteMapView;
-import UI.Art.SpriteArtist.UnitSpriteSet;
+import UI.Art.SpriteArtist.*;
```
Comment (ThatIsAUsername) [inline src/UI/Art/Animation/BaseUnitActionAnimation.java:7]: I'm not sure we approve of this kind of thing around here.

Comment (ThislsAUsername) [reply src/UI/Art/Animation/BaseUnitActionAnimation.java:7]: Why not? It means less churn on the imports, and no annoying warnings after you remove the last reference to something... *like that one we have in Unit ;-;*.

Comment (ThatIsAUsername) [reply src/UI/Art/Animation/BaseUnitActionAnimation.java:7]: It's generally considered bad practice to import with wildcards. It clutters the namespace with a bunch of stuff you aren't using, makes name collisions more likely, and makes it harder to see what you actually use from the package. Plus I just prefer to be explicit about things.

Comment (ThislsAUsername) [reply src/UI/Art/Animation/BaseUnitActionAnimation.java:7]: The clutter is mostly irrelevant... especially in namespaces that *we* define.
Collisions are mostly not a thing we need to worry about, especially if you "be explicit about things" and specify the enclosing scope next to the invocation.

Also, I literally never look at the imports unless the IDE tells me something's wrong with them.
The signal to noise ratio is awful.
Fiddling with that less often is a win in my book.

Comment (ThatIsAUsername) [reply src/UI/Art/Animation/BaseUnitActionAnimation.java:7]: Blah fine. Hopefully we don't find reason to regret it.

**Pull request merged**